cmds/uinit/uinit.go

```diff
@@ -148,6 +148,8 @@ var (
 		util.Symlink{NewPath: "/etc/mtab", Target: "/proc/mounts"},
 		// Sigh.
 		util.Symlink{NewPath: "/bin/sh", Target: "/bin/bash"},
+		// Resolve localhost name
+		util.File{Name: "/etc/hosts", Contents: "127.0.0.1\tlocalhost", Mode: 0644},
```

Comment:
might as well add a \n after localhost and add an IP6 version as well

Reply:
Fixed